html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/EICAS/B747_8_EICAS.js

```diff
@@ -136,49 +126,71 @@ class B747_8_EICAS extends Airliners.BaseEICAS {
             let N2Value = this.getN2Value(i + 1);
             switch (this.engines[i].currentState) {
                 case B747_8_EngineState.IDLE:
-                    if (this.getFuelValveOpen(i + 1)) {
-                        if (N2Value >= this.getN2IdleValue())
-                            this.changeState(i, B747_8_EngineState.RUNNING);
-                        else if (N2Value >= 0.05)
-                            this.changeState(i, B747_8_EngineState.AUTOSTART);
+                if (this.getFuelValveOpen(i + 1)) {
+                    if (N2Value >= this.getN2IdleValue()) {
+                        this.changeState(i, B747_8_EngineState.RUNNING);
                     }
-                    break;
-                case B747_8_EngineState.AUTOSTART:
-                    if (this.getFuelValveOpen(i + 1)) {
-                        if (N2Value >= this.getN2IdleValue())
-                            this.changeState(i, B747_8_EngineState.RUNNING);
+                    else if (N2Value >= 0.05) {
+                        this.changeState(i, B747_8_EngineState.AUTOSTART);
                     }
-                    else {
-                        this.changeState(i, B747_8_EngineState.DECELERATE);
+                }
+                break;
+                case B747_8_EngineState.AUTOSTART:
+                if (this.getFuelValveOpen(i + 1)) {
+                    if (N2Value >= this.getN2IdleValue()) {
```

Comment:
there are still formatting errors here and there unfortunately, this for example is not indented correctly